src/client/public/css/base.css

```diff
@@ -14,11 +14,13 @@
 	--font-size-500: 16px;
 	--font-size-600: 18px;
 	--font-size-700: 20px;
+	--font-size-800: 36px;
```

Comment:
<img width="953" alt="image" src="https://github.com/GEOFARL/online-dictionary/assets/110524091/4ea99abd-1721-4bf6-a38f-74687b772535">

```suggestion
	--font-size-950: 72px;
```